MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -2317,12 +2328,16 @@
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				E27A20D02654EBED00AA935F /* li.tar in Resources */,
 				B4A78D19261FD47700FDF212 /* route-with-banner-instructions.json in Resources */,
 				B4A78D1A261FD47700FDF212 /* route-with-missing-road-classes.json in Resources */,
+				E27A20CF2654EBED00AA935F /* GGPark-to-BernalHeights.route in Resources */,
```

Comment:
The only use of this test fixture appears to have been removed in #785.

Reply:
Thanks, removing.